docs/l3/api_flow.md

```diff
@@ -377,16 +296,30 @@ Response:
 Response:
 ```
 {
-  "id": "606ea493-d761-4446-b456-3e92ecf28931",
+  "id": "d2648206-1e53-40c1-b5af-ec3702b13149",
   "state": "submitted",
-  "localId": "3be18c6e-367f-479e-a5c2-a52aafc6a376",
+  "localId": "6009e0b0-2fa8-48c1-8830-8e250502a9c3",
   "apiType": "match2",
   "transactionType": "proposal",
-  "submittedAt": "2023-05-09T16:22:57.259Z",
-  "updatedAt": "2023-05-09T16:22:57.259Z"
+  "submittedAt": "2023-05-19T11:19:05.361Z",
+  "updatedAt": "2023-05-19T11:19:05.361Z"
 }
 ```
 
+The Optimiser can view the new `match2` on-chain when the block containing it is finalised using `GET /v1/match2/{match2Id}/proposal`.
+
+```
+  {
+    "id": "d2648206-1e53-40c1-b5af-ec3702b13149",
+    "state": "finalised",
+    "localId": "6009e0b0-2fa8-48c1-8830-8e250502a9c3",
+    "apiType": "match2",
+    "transactionType": "proposal",
+    "submittedAt": "2023-05-19T11:19:05.361Z",
+    "updatedAt": "2023-05-19T11:19:19.853Z"
+  }
+```
```

Comment:
Text says `can view the new match2...using GET /v1/match2/{match2Id}/proposal` but that route lists proposal transactions rather than returning a `match2`. Is the purpose of the section to show a finalised transaction or to show a `match2`? If it's to show a finalised transaction then maybe the first line should say something like `...containing it is finalised. The status of the transaction can be returned with GET /v1/match2/{match2Id}/proposal/{proposalId}`

Reply:
Yeah that's right, wasn't quite accurately described.